### PR TITLE
Prof: Setup organisation page container

### PR DESCRIPTION
**Pull request by chidioguejiofor**
#### What does this PR achieve?
- adds settings page to the app
- adds organisation portal to the app
- updates storybook docs for objects changed
- adds settings page to the app
- adds reducers, sagas and actions for settings
- makes calls to API and displays the data to the user
- updates storybook with new changes
- adds routes for retrieving data for settings to the app
- ensures that the feature works well

#### Any background context?
- This PR does not implement the searching functionally yet. This should be in a separate PR

#### Manual Testing Steps
- Login
- Extract one of your organisation Id from 

#### Relevant Trello Tickets
- [Organisation Portal](https://trello.com/c/uFCORWns/105-create-organisation-portal-container)

### CODE REVIEW

File: src/store/reducers/setting.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      };
+    case settingsConstants.FETCH_APPLIANCE_CATEGORY_SUCCESS:
+      return {
+        ...state,
+        applianceCategory: {
+          ...state.applianceCategory,
+          fetching: false,
+          fetched: true,
+          data: payload.data,
+          meta: payload.meta,
```
Comment (chineduknight) [inline src/store/reducers/setting.ts]: if its possible I think you can destructure the items needed from the payload

File: src/store/sagas/setting.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  yield takeLatest(settingsConstants.FETCH_APPLIANCE_CATEGORY_REQUEST, callFetchApplianceCategory);
+}
+
+export function* watchFetchParameters() {
+  yield takeLatest(settingsConstants.FETCH_PARAMETERS_REQUEST, callFetchParameters);
+}
+
+export default function* settingsSaga() {
+  yield all([fork(watchFetchApplianceCategory), fork(watchFetchUnits), fork(watchFetchParameters)]);
+}
```
Comment (chineduknight) [inline src/store/sagas/setting.ts]: I think all these can be grouped

File: src/store/actions/setting/index.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import * as settingsConstants from './constants';
+
+export const fetchApplianceCategory = (payload): Action => ({
+  type: settingsConstants.FETCH_APPLIANCE_CATEGORY_REQUEST,
+  payload,
+});
+
+export const fetchParameters = (payload): Action => ({
+  type: settingsConstants.FETCH_PARAMETERS_REQUEST,
+  payload,
```
Comment (chineduknight) [inline src/store/actions/setting/index.ts]: Destructuring here would be nice as well in order not to  repeat settingsConstants

File: src/components/ui/Table/index.tsx [hunk 4]
```diff
@@ -26,7 +33,7 @@ TableItem.Wrapper = styled.div<Pick<ItemProps, 'flexValue' | 'textAlign'>>`
   padding-left: 2%;
   width: 100%;
 `;
-TableCard.Wrapper = styled.div`
+TableCard.Wrapper = styled.div<Pick<ItemProps, 'mouseCursor'>>`
   display: flex;
   height: 50px;
   background-color: ${SECONDARY_LIGHT_SMOKE};
```
Comment (chineduknight) [inline src/components/ui/Table/index.tsx]: pixel value I thought we were using rem/em

File: src/components/pages/OrganisationPortal/Settings/index.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+      />
+    </NavigationLayout>
+  );
+}
+
+const mapStateToProps = state => ({
+  applianceCategory: state.setting.applianceCategory,
+  parameters: state.setting.parameters,
+  units: state.setting.units,
+});
```
Comment (chineduknight) [inline src/components/pages/OrganisationPortal/Settings/index.tsx]: destructure settings from state

**Review by chineduknight — APPROVED** [review]
Minor changes.
Great Job with the sagas
LGTM

File: src/components/pages/OrganisationPortal/Settings/index.tsx (hunk 5 above)
Comment (chidioguejiofor) [reply src/components/pages/OrganisationPortal/Settings/index.tsx]: Nice

File: src/store/actions/setting/index.ts (hunk 3 above)
Comment (chidioguejiofor) [reply src/store/actions/setting/index.ts]: Nice one.

File: src/store/sagas/setting.ts (hunk 2 above)
Comment (chidioguejiofor) [reply src/store/sagas/setting.ts]: Nice didn't know this was possible until now. Thanks for that

File: src/components/ui/Table/index.tsx (hunk 4 above)
Comment (chidioguejiofor) [reply src/components/ui/Table/index.tsx]: Oversight  👀

**Review by chidioguejiofor — COMMENTED** [review]
Responded to feedback. Ready for merge

**Pull request merged**